apps/examples/rdp/README.txt

```diff
@@ -0,0 +1,11 @@
+examples/rdp
+^^^^^^^^^^^^^^
+
+  Read Protect (RDP) is used to protect high security level code in AmebaD
```

Comment:
I agree Mr Chang.
The Secure fw consists of components from os (which are standard) and components from soc (which are specific to a particular board or implementation)
Since the example uses both, I will think of some way to divide them in separate config dependencies